src/shared-components/optionButton/index.tsx

```diff
@@ -12,27 +12,48 @@ import {
 } from './style';
 import CheckmarkIcon from '../../svgs/icons/checkmark-icon.svg';
 
+export type OptionButtonProps = {
+  buttonType?: 'primary' | 'secondary';
```

Comment:
I like the updated naming - this is pretty consistent with other components as well, I believe.